app/views/internal/configs/show.html.erb

```diff
@@ -293,6 +293,16 @@
               </div>
             </div>
 
+            <div class="form-group">
+              <%= f.label :suggested_users %>
+              <%= f.text_field :suggested_users,
+                               class: "form-control",
+                               value: SiteConfig.suggested_users.join(","),
+                               placeholder: "List of valid usernames: comma separated, letters only e.g. beginners,javascript,ruby,swift,kotlin" %>
+              <div class="alert alert-info">Determines which users are suggested to follow to new users during onboarding (comma
```

Comment:
```
Determines which users are suggested to follow to new users during onboarding (comma separated, letters only). Please note that these users will be shown as a fallback if no recently-active commenters or producers can be suggested
```

I feel like we should try to keep these descriptions to be shorter and more direct, and so I took a suggestive stab at trying to capture your points in the sentence below. I removed `comma separated, letters only` as its a repetition from the placeholder.

```
Determines suggested followers to new users during onboarding. N.B. These will be your fallback users if no recently-active content creators can be suggested
```

Reply:
@Ridhwana I’m open to changing this but personally I don’t know what “N.B.” is, so I find this more confusing.

Reply:
Ah, N.B is Note well, and is more commonly used as part of British English, which is what I speak 😁 let's keep it as please note instead of N.B then :)

Reply:
Ahhh makes much more sense now. I’ll try to rework this. I do worry that if we don’t mention the “comma separated” but that some admins might not know that this is how to list the usernames though. Is that not a valid concern?

Reply:
Hmmm so the use case for it would mostly likely be if there's only a single item in the list. I think let's just keep it 👍🏽

Reply:
Okay. I’m going to leave this sentence as-is in that case! 👍

Reply:
Trivia: we use N.B. in Italian as well, it's the latin `Nota Bene` https://en.wikipedia.org/wiki/Nota_bene

Reply:
@Ridhwana @rhymes I'm gonna start using this and trip up all my American English speaking friends 😂